### PR TITLE
Add Code of Conduct

### PR DESCRIPTION
**Pull request by SoManyHs**

### CODE REVIEW

File: code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+* Publishing other's private information, such as physical or electronic addresses, without explicit permission
+* Other unethical or unprofessional conduct
+
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct. By adopting this Code of Conduct, project maintainers commit themselves to fairly and consistently applying these principles to every aspect of managing this project. Project maintainers who do not follow or enforce the Code of Conduct may be permanently removed from the project team.
+
+This code of conduct applies both within project spaces and in public spaces when an individual is representing the project or its community.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by opening an issue or contacting one or more of the project maintainers.
+
+This Code of Conduct is adapted from the Contributor Covenant, version 1.2.0, available from http://contributor-covenant.org/version/1/2/0/.
```
Comment (aaithal) [inline code-of-conduct.md]: Can we use the latest one? https://contributor-covenant.org/version/1/4/

Comment (SoManyHs) [reply code-of-conduct.md]: @simplycloud is running this by the AWS OSS legal team and should have an update shortly.

**Comment by SoManyHs** [comment]
@simplycloud should have an update to this shortly.

**Comment by simplycloud** [comment]
Please refer to https://github.com/aws/amazon-ecs-cli/pull/377 for latest CoC contribution.

**Pull request closed without merging**